### ISSUE TITLE
ngram regression

### ISSUE THREAD

**Issue by stevenbird** [nltk/nltk#1194]
The CI server reports errors such as the following for the ngram functions (build [456](https://nltk.ci.cloudbees.com/job/nltk/TOXENV=py27-jenkins,jdk=jdk8latestOnlineInstall/456/) following merge of #1179).

```
Failed example:
    list(bigrams([1,2,3,4,5]))
Exception raised:
    Traceback (most recent call last):
      File "<doctest nltk.util.bigrams[1]>", line 1, in <module>
        list(bigrams([1,2,3,4,5]))
      File "/scratch/jenkins/workspace/nltk/TOXENV/py27-jenkins/jdk/jdk8latestOnlineInstall/nltk/util.py", line 483, in bigrams
        for item in ngrams(sequence, 2, **kwargs):
      File "/scratch/jenkins/workspace/nltk/TOXENV/py27-jenkins/jdk/jdk8latestOnlineInstall/nltk/util.py", line 460, in ngrams
        history.append(next(sequence))
    TypeError: list object is not an iterator
```

**Comment by stevenbird** [comment]
@alvations – would you be able to have a look at this please?

**Comment by alvations** [comment]
@stevenbird , sorry for missing that out in the previous merge. The fix #1195  removes the checks and converts into iterable when padding.

**Comment by stevenbird** [comment]
Thanks @alvations
